python/cudf/cudf/core/column/string.py

```diff
@@ -3650,7 +3650,12 @@ def isempty(self) -> SeriesOrIndex:
         4    False
         dtype: bool
         """
-        return self._return_or_inplace((self._column == "").fillna(False))
+        return self._return_or_inplace(
+            # mypy can't deduce that the return value of
```

Comment:
Can we use a “cast” instead of ignoring the type?

Reply:
Done.